sort_data.py

```diff
@@ -4,6 +4,11 @@
 
 import pytoml
 
+header = """# The schema for this file can be found at https://github.com/rust-gamedev/arewegameyet#contribute.
```

Comment:
Yeah, this feels somewhat too hacky to me

Reply:
Reverted that bit of the change for now.

Do you think it'd be better to have the page link to the editor, or to just link to the contributing info?

Reply:
A link to the contributing info should be enough, I guess :)

Reply:
I'll switch the link now, then, and maybe add a note to the contributing guidelines saying that you can edit via GitHub rather than pulling a local copy.

Reply:
Oh, duh, it already mentions that 😅

Reply:
Done :+1: